core/embed/extmod/modtrezorutils/modtrezorutils-meminfo.h

```diff
@@ -487,6 +493,14 @@ void dump_trezor_vcp(FILE *out, const mp_obj_VCP_t *vcp) {
   fprintf(out, ",\n");
 }
 
+void dump_protomsg(FILE *out, const mp_obj_protomsg_t *value) {
+  print_type(out, "protomsg", NULL, value, false);
+  fprintf(out, ",\n\"shortval\": ");
+  dump_short(out, protobuf_debug_name(value));
```

Comment:
i would think something like this could work, instead of defining a function?
```
mp_obj_t name[2] = {MP_OBJ_NULL, MP_OBJ_NULL};
mp_obj_type_t *type = mp_obj_get_type(value); // or protobuf_debug_msgtype();
type->attr(value, MP_QSTR_MESSAGE_NAME, name);
dump_short(out, name[0]);
```

Reply:
nice trick, fixed in 88d5f81